source/renderer/app/components/wallet/transactions/FilterButton.scss

```diff
@@ -1,29 +1,8 @@
 .component {
-  position: absolute;
-  right: 10px;
-  top: 10px;
-  z-index: 98;
-
   .actionButton {
-    border-radius: 5px;
-    box-shadow: 0 1.5px 5px 0
-      var(--theme-transactions-filter-button-shadow-color);
+    border-radius: 2px;
```

Comment:
@a-rukin what is the radius we need for the Export/Filter buttons?

Reply:
@a-rukin @daniloprates this is what I mean - we need to keep these buttons consistent across different screens:
![transactions-vs-rewards](https://user-images.githubusercontent.com/376611/98931459-573eb880-24de-11eb-8ebc-97fb747e22b9.png)

Reply:
top styles are correct
btw dates mini-titles (tx screen) would be removed later?

Reply:
OK so we need to update the styles of the "Export CSV" button on the "Rewards" screen @daniloprates 🙏 

> btw dates mini-titles (tx screen) would be removed later?

I don't recall what was the decision on this one ☝️

Reply:
important that "earned delegation rewards + export button" should be the same sticky header as transactions 
ofc in case you are going to unify them 
cc @nikolaglumac @daniloprates

Reply:
That can be done in a separate task. @a-rukin please create one for that and add a design for that very states...
@daniloprates please only adjust border radius in this PR.

Reply:
it was updated month ago https://zpl.io/VOBk15Y

Reply:
https://jira.iohk.io/browse/DDW-453

Reply:
Thanks @a-rukin! 

@daniloprates please adjust only the border-radius of the button in the scope of this PR 🙏

Reply:
sorry to be difficult again, but in case you gonna implement fix here pay attention to:
you need still to keep shadow here (as it is still FAB)
but shadow would be removed in next PR